source/renderer/app/utils/formatters.js

```diff
@@ -19,7 +19,8 @@ import type { AssetMetadata } from '../api/assets/types';
 export const formattedWalletAmount = (
   amount: BigNumber,
   withCurrency: boolean = true,
-  long: boolean = true
+  long: boolean = true,
+  currency: string = 'ADA'
```

Comment:
Again, I think it should be lowercase, if yes take it from WalletUnits

Reply:
This should be upper case as it's the currency symbol. Also the function body already had `ADA` hard coded.